src/scenes/Wallet/components/Balances/styles.js

```diff
@@ -0,0 +1,62 @@
+const styles = (theme) => ({
+  rootPaper: {
```

Comment:
try using a more descriptive name like balancePaper. Even though we don't have the scope problem with JSS but we want the code to be more readable and can identify a component at first glance.

Reply:
renamed. in this case, should we start off all styles with `myBalanceComponentName`?